### PR TITLE
MakeSlippery proc made more readable with named arguments

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Comment by optimumtact** [comment]
@phil235 @nullbear

**Comment by freerealestate** [comment]
what

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -95,10 +95,9 @@
 					C.spin(1,1)
 		return 1
 
-/turf/open/proc/MakeSlippery(wet_setting = TURF_WET_WATER, min = 0, max = 0) // 1 = Water, 2 = Lube, 3 = Ice
-	wet_time += max
-	if(wet_time < min)
-		wet_time = min
+/turf/open/proc/MakeSlippery(wet_setting = TURF_WET_WATER, min_wet_time = 0, wet_time_to_add = 0) // 1 = Water, 2 = Lube, 3 = Ice
+	wet_time += wet_time_to_add
+	wet_time = max(wet_time, min_wet_time)
```
Comment (RemieRichards) [inline code/game/turfs/open.dm]: ```wet_time = max(wet_time + wet_time_to_add, min_wet_time)```

Comment (freerealestate) [inline code/game/turfs/open.dm]: wet_time = max(wet_time + wet_time_to_add, min_wet_time)wet_time = max(wet_time + wet_time_to_add, min_wet_time)wet_time = max(wet_time + wet_time_to_add, min_wet_time)

**Comment by phil235** [comment]
You need to change all proc calls that specify the old arg names. See https://github.com/tgstation/-tg-station/pull/17333/files#diff-a6ca063715b3e16dcfee3a493a8a4aa9R177 for example

**Comment by optimumtact** [comment]
so I will indeed

**Comment by optimumtact** [comment]
Ready to go

**Comment by nullbear** [comment]
thanks! Been busy with life stuff

**Pull request merged**